src/matchers/toBeEmpty/predicate.test.js

```diff
@@ -1,74 +0,0 @@
-import { equals } from 'expect/build/jasmineUtils';
-import predicate from './predicate';
-
-describe('toBeEmpty Predicate', () => {
-  describe('returns true', () => {
-    test('When empty string is passed', () => {
-      expect(predicate(equals, '')).toBe(true);
-    });
-
-    test('When empty string object is passed', () => {
-      expect(predicate(equals, new String(''))).toBe(true);
-    });
-
-    test('When empty array is passed', () => {
-      expect(predicate(equals, [])).toBe(true);
-    });
-
-    test('When empty object is passed', () => {
-      expect(predicate(equals, {})).toBe(true);
-    });
-
-    test('When empty Set is passed', () => {
```

Comment:
Set and map tests aren't in _index.test.js_. Not sure if we care about that.

Reply:
see https://github.com/jest-community/jest-extended/pull/410/commits/06c805168ae4c861e8c90084404ba43a115734eb